content/en/docs/aws/pipeline.md

```diff
@@ -47,7 +47,7 @@ client.list_experiments(namespace="<your_namespace>")
 
 - dex {{% config-uri-aws-standard %}}
 
-Please look at this [PR](https://github.com/kubeflow/kfctl/issues/140#issuecomment-578837304) to do programmatic authentication with Dex.
+Please look at [this comment](https://github.com/kubeflow/kfctl/issues/140#issuecomment-578837304) and [this comment](https://github.com/kubeflow/kfctl/issues/140#issuecomment-719894529) to do programmatic authentication with Dex.
```

Comment:
Hey @yanp Thanks for the update  

I'd recommend to reword a little:

```suggestion
To do programmatic authentication with Dex, refer to the following comments under the [#140](https://github.com/kubeflow/kfctl/issues/140) issue in the `kfctl` repository: [#140 (comment)](https://github.com/kubeflow/kfctl/issues/140#issuecomment-578837304) and [#140 (comment)](https://github.com/kubeflow/kfctl/issues/140#issuecomment-719894529).
```

Can you also fix the capitalization ("AWS") in line 87?

```diff
- If you write any files to S3 in your application, use `use_aws_secret` to attach aws secret to access S3.
+ If you write any files to S3 in your application, use `use_aws_secret` to attach an AWS secret to access S3.

```

Thank you!

/assign @Jeffwan @PatrickXYS

(AWS team)